rubygem-foreman_azure/rubygem-foreman_azure.spec

```diff
@@ -0,0 +1,79 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_azure
+
+Summary: Azure integration with Foreman (theforeman.org)
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.0.1
+Release: 1%{?foremandist}%{?dist}
+Group:   Applications/System
+License: GPLv3
+URL:     https://github.com/theforeman/foreman_azure
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+Requires: foreman >= 1.9.0
```

Comment:
Oh, I compared it to your Debian PR which stated >= 1.11.0.  The engine.rb actually states >= 1.8.